src/main/java/nextstep/subway/application/SectionService.java

```diff
@@ -0,0 +1,55 @@
+package nextstep.subway.application;
+
+import nextstep.subway.domain.*;
+import nextstep.subway.dto.SectionRequest;
+import nextstep.subway.dto.SectionResponse;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+@Transactional(readOnly = true)
+public class SectionService {
+
+    private LineRepository lineRepository;
+    private StationRepository stationRepository;
+    private SectionRepository sectionRepository;
+
+    private static final String NO_SUCH_LINE_EXCEPTION = "해당 ID의 노선 정보가 없습니다.";
+    private static final String NO_SUCH_STATION_EXCEPTION = "해당 ID의 역 정보가 없습니다.";
+
+    public SectionService(LineRepository lineRepository, StationRepository stationRepository, SectionRepository sectionRepository) {
+        this.lineRepository = lineRepository;
+        this.stationRepository = stationRepository;
+        this.sectionRepository = sectionRepository;
+    }
+
+    @Transactional
+    public List<SectionResponse> saveSection(Long lineId, SectionRequest sectionRequest) {
+        Line line = findLineById(lineId);
+        Station upStation = findStationById(sectionRequest.getUpStationId());
+        Station downStation = findStationById(sectionRequest.getDownStationId());
+
+        Sections sections = line.addSection(sectionRequest.toSection(line, upStation, downStation, sectionRequest.getDistance()));
```

Comment:
```suggestion
        Sections sections = line.addSection(sectionRequest.toSection(line, upStation, downStation));
```
`distance`는 `sectionRequest`에 포함된 값이니까 파라미터를 생략할수 있지 않을까요?

Reply:
넵! 수정했습니다~🙇‍♂️